addon/kubeflow-operator/manifest.yaml

```diff
@@ -0,0 +1,472 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
@rastislavs I was thinking about splitting the entire manifest.yaml file into 2 files, one for kubeflow operator, and one for KfDef, is there a way for us to make sure that addon controller can first apply the operator manifest and then the KfDef?

Reply:
God idea... It seems that reading the manifests goes down to this func: https://github.com/kubermatic/kubermatic/blob/master/pkg/addon/template.go#L187

"ReadDir reads the directory named by dirname and returns a list of directory entries sorted by filename."

So we could control the order by conscious naming :)

Reply:
Nice to know that :)
How do we keep the order and find meaningful names at the same time?
I was thinking about something like:
```
- operator.yaml
- kfdef.yaml
```
but the order is wrong...
Also, I am not sure if we should depend on the order in addon implementation, I just don't know if they will change anything in the future, which will break our addon :)

Reply:
Right, let's keep it like it is for now.. Maybe we could solve this common problem with CRDs on the addon controller level first.

Reply:
Agree :)